lib/executor.js

```diff
@@ -168,23 +168,24 @@ function setupModels(app, instructions) {
 }
 
 function defineModels(app, instructions) {
+  var registry = app.registry || app.loopback;
```

Comment:
So far, loopback-boot was not tied to a specific version of loopback, one could use loopback-boot 2.x with loopback 1.x and vice versa. This feature allows developer to split upgrade of loopback runtime and project structure into two smaller independent steps.

I would like to preserve this feature for as long as the maintenance costs are low. Therefore I don't want to drop this part at the time of LoopBack 3.0 release.